### PR TITLE
Feature/10876 add new info box on person screen

### PR DESCRIPTION
**Pull request by 30mar**
## Description
Add new info box for admission state

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10876

## Screenshots
IMG_10D97D643587-1

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/Cells/AdmissionState/AdmissionStateCellModel.swift [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import Foundation
+import OpenCombine
+import UIKit.UIFont
+
+final class AdmissionStateCellModel {
+
+	// MARK: - Init
+
+	init(
+		healthCertifiedPerson: HealthCertifiedPerson
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/Cells/AdmissionState/AdmissionStateCellModel.swift:14]: Should be enough to inject the the admission state, no need to inject the whole person

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/HealthCertifiedPersonViewModel.swift [hunk 2]
```diff
@@ -203,6 +209,9 @@ final class HealthCertifiedPersonViewModel {
 	private var subscriptions = Set<AnyCancellable>()
 
 	private var healthCertificateCellViewModels = [HealthCertificateCellViewModel]()
+	private var admissionStateIsVisible: Bool {
+		return !(healthCertifiedPerson.admissionState == .other)
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/HealthCertifiedPersonViewModel.swift:213]:
```suggestion
		return healthCertifiedPerson.admissionState != .other
```

**Review by nickguendling — APPROVED** [review]
Just two small comments, nothing major

**Review by CV113 — COMMENTED** [review]
We are currently discussing some language formulations, please do not merge them yet!

**Review by marcussc — APPROVED** [review]

**Comment by 30mar** [comment]
@CV113 We need to merge the ticket so we don't block our development, whenever the text is ready please create a ticket and we will get right on it

**Pull request merged**